packages/back-office-service/prisma/migrations/20230316101542_init/migration.sql

```diff
@@ -0,0 +1,74 @@
+BEGIN TRY
+
+BEGIN TRAN;
+
+-- CreateTable
+CREATE TABLE [dbo].[Project] (
+    [id] INT NOT NULL IDENTITY(1,1),
+    [caseId] INT NOT NULL,
+    [caseReference] NVARCHAR(1000),
```

Comment:
id, caseId and caseRef ?

Reply:
`id` is possibly not needed but `caseId` and `caseReference` come from Back Office. There is some difference between those right @Philip-Philippou-KC ?

we could potentially use `caseId` as our primary key instead of having our own internal `id` field. I think in reality, we're not going to use `id`, we'll be querying by `caseReference` (or maybe `caseId`)

Reply:
We would need our own synthetic id for joining etc. If caseId is the BO office synthetic id, then we should not store that (and really the BO should not leak this out in its messages)

Reply:
The caseId is BO DB auto-increment Id.  I think their API accesses the resources in their endpoints using that Id (need to confirm that).  All of our existing endpoints in our API use the caseReference so we need that.  We may need all 3, id for our internal joins, integrations with BO may need caseId for linkage, caseRef to satisfy our own API requirements.